content/docs/Quick starts/quick-start-linux.md

```diff
@@ -27,6 +27,7 @@ You must have the following software installed on your machine:
 *   Python version 2.7 or greater
 *   The Python [`pip` command](https://pypi.org/project/pip/)
 *   The Python [virtualenv tool](https://pypi.org/project/virtualenv/)
+*   The ```python-dev``` (Ubuntu/Debian) or ```python-devel```(Fedora/Cent-OS) package 
```

Comment:
Generally it's best to avoid parentheses as readers tend to ignore information in them. You could rewrite this bullet point something like:

*   If you're using Ubuntu or Debian, the `python-dev` package
*   If you're using Fedora or Cent-OS, the `python-devel` package

Reply:
Thanks, I have reformatted it conforming to the said format, please verify it.